src/app/components/play-round/play-round.component.html

```diff
@@ -23,8 +23,8 @@
   <div class="marker-name">
     {{marker.name | titlecase}}
   </div>
-  <div class="nest-guide" *ngIf="marker.isNest && !marker.canVisit">
-    Gather Pollen to deposit them in your nest
+  <div class="marker-tip" *ngIf="showTip(marker) as tip">
```

Comment:
Don't use functions in data binding. The tip and logic for creating it should be moved to the observables for the markers.